internal/rollbar/rollbar.go

```diff
@@ -91,6 +90,15 @@ func SetupRollbar(token string) {
 		// We're not a server, so don't send server info (could contain sensitive info, like hostname)
 		data["server"] = map[string]interface{}{}
 		data["platform_os"] = runtime.GOOS
+		if _, exists := data["request"]; !exists {
+			data["request"] = map[string]string{}
+		}
+		if request, ok := data["request"].(map[string]string); ok {
+			// Rollbar specially interprets the body["request"]["user_ip"] key, but it does not have to
+			// actually be an IP address. Use device_id for now. We may want to use real IP later if it
+			// is easily accessible (Rollbar does not provide it for us with non-http.Request errors).
+			request["user_ip"] = uniqid.Text()
+		}
```

Comment:
```suggestion
		// Rollbar specially interprets the body["request"]["user_ip"] key, but it does not have to
		// actually be an IP address. Use device_id for now. We may want to use real IP later if it
		// is easily accessible (Rollbar does not provide it for us with non-http.Request errors).
		data["request"]["user_ip"] = uniqid.Text()
```

Doesn't seem like the check serves a purpose anymore due to the line 93 addition.

Reply:
The check is required. It's a compile error otherwise because data is a `map[string]interface{}`.